go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/mesg-foundation/core
+module github.com/mesg-foundation/engine
```

Comment:
This breaks everything because it's not in sync with what Github has or something like that, @NicolasMahe can explain more but I this that's why everything fails 
```suggestion
module github.com/mesg-foundation/core
```

Reply:
@NicolasMahe how it brakes,  what is not sync I don't get it.

Reply:
I will check this afternoon ;)

Reply:
it was because the `go.mod` in github.com/mesg-foundation/engine is still with `module github.com/mesg-foundation/core`.
So `go mod install` was returning the error `go.mod path is different than source` or something like this.

Reply:
I guess same problem as docker face by renaming to moby. All the go sources are still under docker repo to not break previous version i think

Reply:
I will check in depth in a few hours

Reply:
oh ok, i guess import names was never changed, i'll update this and check